FavroPlasticExtension/FavroExtension.cs

```diff
@@ -140,18 +184,24 @@ public List<PlasticTask> LoadTasks(List<string> taskIds)
 
         public List<PlasticTask> GetPendingTasks()
```

Comment:
It may be more standard to search by assigned but not completed, no?

Reply:
Ok, and doing this also I can remove the TODO column configuration from preferences.